### ISSUE TITLE
Missing Documentation for this wrapper

### ISSUE THREAD

**Issue by kartik9k** [redbear/nRF5x#40]
Hey! Thanks to readbear for providing this wrapper to program BLEs using Arduino IDE. However, I believe that this project lacks proper documentation so that it can be used easily.

Please let me know if it is present and I've missed it completely, otherwise please provide one soon! :)

**Comment by Ryanauger95** [comment]
@kartik9k Did you find any documentation outside of the examples?

**Comment by kartik9k** [comment]
Nope! I finally used this library: https://github.com/sandeepmistry/arduino-BLEPeripheral
